web/portal/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/page/PageConfigOptions.groovy

```diff
@@ -49,5 +49,9 @@ mixPageConfigs.addSelectItemOption(new SelectItemOption("mixPage.TwoColumnsOneRo
 mixPageConfigs.addSelectItemOption(new SelectItemOption("mixPage.OneRowTwoColumnsLayout", "one-row-two-columns", "OneRowTwoColumnsLayout"));
 mixPageConfigs.addSelectItemOption(new SelectItemOption("mixPage.ThreeRowsTwoColumnsLayout", "three-rows-two-columns", "ThreeRowsTwoColumnsLayout"));
 
+SelectItemCategory analyticsPageConfigs = new SelectItemCategory("analyticsPageConfigs") ;
+categories.add(analyticsPageConfigs);
+analyticsPageConfigs.addSelectItemOption(new SelectItemOption("analyticsPage.AnalyticsLayout", "analytics", "AnalyticsLayout"));

```

Comment:
We will need to rework the Page Templates extensibility system to use Component plugins as any extensible behavior in the platform, especially to avoid adding analytics in gatein-portal layer